README.rst

```diff
@@ -244,10 +244,8 @@ Starting a new Mars on Ray runtime locally via:
 
 .. code-block:: python
 
-    import ray
-    ray.init()
     import mars
-    mars.new_ray_session(worker_num=2)
+    mars.new_session(backend='ray')
```

Comment:
Maybe both oscar and ray task mode should be documented here?

Reply:
> 

Too many modes confuse users. So, we only recommend the Ray DAG mode.